src/types.ts

```diff
@@ -5,6 +5,7 @@ export interface Script {
     defer?: boolean;
     async?: boolean;
     crossOrigin?: '' | 'anonymous' | 'use-credentials';
+    type?: 'importmap' | 'module' | (string & {});
```

Comment:
could just `type?: 'importmap' | 'module' | string` be enough?

Reply:
> could just `type?: 'importmap' | 'module' | string` be enough?

typescript reduces this type to just `string` and autocomplete does not work with it.